wolfcrypt/src/aes.c

```diff
@@ -6702,9 +6716,33 @@ int wc_AesGcmSetKey(Aes* aes, const byte* key, word32 len)
         VECTOR_REGISTERS_POP;
     }
     if (ret == 0) {
-    #if defined(GCM_TABLE) || defined(GCM_TABLE_4BIT)
-        GenerateM0(&aes->gcm);
-    #endif /* GCM_TABLE */
+#if defined(GCM_TABLE) || defined(GCM_TABLE_4BIT)
```

Comment:
Do we have a way to hit this change with our benchmark tool?

Reply:
Made change to benchmark: -aead_set_key.
./configure '--disable-shared' '--enable-intelasm' '--enable-aesgcm-stream' 'CFLAGS=-DWC_C_DYNAMIC_FALLBACK'
./configure '--disable-shared' '--enable-intelasm' '--enable-aesgcm-stream''
./wolfcrypt/benchmark/benchmark -aead_set_key -aes-gcm 16

Code before change slowest
Dynamic fallback faster.
No dynamic fallback faster.